autoload/neomake/makers/ft/jade.vim

```diff
@@ -0,0 +1,18 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#jade#EnabledMakers()
+    let makers = []
+    if neomake#utils#Exists('jade-lint')
```

Comment:
does it check for the name of the maker or the name of the "exe" arg?
(because the maker is named "jadelint" while the command is "jade-lint")